DigitalLearningSolutions.Data/DataServices/SelfAssessmentDataService/CandidateAssessmentsDataService.cs

```diff
@@ -142,6 +145,39 @@ public void UpdateLastAccessed(int selfAssessmentId, int delegateUserId)
             }
         }
 
+        public void RemoveSignoffRequests(int selfAssessmentId, int delegateUserId, int competencyGroupId)
+        {
+            var candidateAssessmentSupervisorVerificationsId = connection.QueryFirst<int>(
+                @" SELECT casv.ID 
+                    FROM( SELECT DISTINCT casv.* FROM CandidateAssessmentSupervisorVerifications AS casv
+                    INNER JOIN CandidateAssessmentSupervisors AS cas
+                        ON casv.CandidateAssessmentSupervisorID = cas.ID
+                    INNER JOIN CandidateAssessments AS ca
+                        ON cas.CandidateAssessmentID = ca.ID
+                    INNER JOIN SupervisorDelegates AS sd
+                        ON cas.SupervisorDelegateId = sd.ID
+                    INNER JOIN AdminUsers AS au
+                        ON sd.SupervisorAdminID = au.AdminID
+                    LEFT OUTER JOIN SelfAssessmentSupervisorRoles AS sasr
+                        ON cas.SelfAssessmentSupervisorRoleID = sasr.ID
+						  INNER JOIN SelfAssessmentStructure AS SAS 
+						  ON CA.SelfAssessmentID = SAS.SelfAssessmentID
+                    INNER JOIN CompetencyGroups AS CG 
+					ON SAS.CompetencyGroupID = CG.ID AND SAS.SelfAssessmentID =@selfAssessmentId
+                    WHERE ((ca.DelegateUserID = @delegateUserId) AND (ca.SelfAssessmentID = @selfAssessmentId) AND (sasr.SelfAssessmentReview = 1) AND (CG.ID =@competencyGroupId) AND (casv.SignedOff =0)

```

Comment:
Please use the braces correctly. See the sample provided by Sarathlal - we need to segregate the bunch of ANDs from the OR. Like 
( (xyz) AND (xyz) AND (xyz) ) OR ( (xyz) AND (xyz) AND (xyz) )

Reply:
Done